app/lib/methods/helpers/addProtocol.ts

```diff
@@ -0,0 +1,8 @@
+const addProtocol = (url: string): string => {
+	if (!url.toLowerCase().startsWith('http')) {
+		return `https://${url.replace('//', '')}`;
```

Comment:
How you know if there's a certificate on the URL?
This is going to fail on localhost envs, for example.

Reply:
the server didn't return the localhost as a link if the user writes only `localhost:3000`, is it expected to return as a link, or should I be able to treat this behavior too?
![Screen Shot 2023-02-16 at 13 57 14](https://user-images.githubusercontent.com/47038980/219434715-8fdfd6a8-0869-4fc4-a632-83b60cccf695.png)

Reply:
I don't understand what you're trying to do. Can you elaborate?

Reply:
I found this link with `http` protocol, that I think it's secure to test. http://web.simmons.edu/~grovesd/comm244/notes/week2/links

you can send it on the chat without the protocol, the behavior will be the same while using the browser or the mobile app

Reply:
> I don't understand what you're trying to do. Can you elaborate?

About my last answer before your question, 
> the server didn't return the localhost as a link if the user writes only localhost:3000, is it expected to return as a link, or should I be able to treat this behavior too?

I explained that the server doesn't consider the localhost:3000 as a link.

Also, we aren't able to check if there's a security certificate or not from a link without their protocol, so as the `https` is a convention to use, we add this protocol at the beginning of the link.